bindings/kepler.gl-jupyter/js/lib/keplergl/components/panel-header.js

```diff
@@ -36,6 +39,12 @@ export function CustomPanelHeaderFactory() {
       blank: true,
       tooltip: 'tooltip.documentation',
       onClick: () => {}
+    },
+    {
+      ...defaultActionItems.find(item => item.id === 'save'),
+      label: null,
+      tooltip: 'Save Map',
+      onClick: () => exportJson({visState: props.visState})
```

Comment:
why only export `visState`? why not the entire kepler.gl state?

Reply:
The export function only uses `visState`. However, the only reason I did it like this is because I copied a `visStateLens` line from elsewhere. Is this [keplerGlReducerCore](https://github.com/keplergl/kepler.gl/blob/master/src/reducers/index.js#L27) (default export) the entire store as a lens?